RadarIO.nuspec

```diff
@@ -17,13 +17,6 @@
 		<tags>radar, radario, xamarin, maui, android, ios, plugin for xamarin, geocoding, radar, geolocation, geofencing, background geolocation, location tracking</tags>
 		<dependencies>
 			<group targetFramework="netstandard2.0"></group>
-			<group targetFramework="MonoAndroid12.0">
-				<dependency id="Xamarin.Kotlin.StdLib" version="1.8.20.1" />
-				<dependency id="Xamarin.GooglePlayServices.Location" version="121.0.1.1" />
-				<dependency id="Xamarin.GooglePlayServices.Ads.Identifier" version="118.0.1.3" />
-				<dependency id="Xamarin.Google.Android.Play.Integrity" version="1.3.0.1" />
-			</group>
-			<group targetFramework="xamarinios10"></group>
```

Comment:
Why are we removing these dependencies? Are they included elsewhere now?

Reply:
We're dropping xamarin because it's EOL